.idea/workspace.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file. I'd be OK with adding `.idea` to `.gitignore` though, if that makes things easier for you.